src/bacnet/basic/object/device.c

```diff
@@ -515,6 +515,7 @@ static BACNET_DEVICE_STATUS System_Status = STATUS_OPERATIONAL;
 static char *Vendor_Name = BACNET_VENDOR_NAME;
 static uint16_t Vendor_Identifier = BACNET_VENDOR_ID;
 static char Model_Name[MAX_DEV_MOD_LEN + 1] = "GNU";
+static char Serial_Number[MAX_DEV_MOD_LEN + 1];
```

Comment:
initialise it to "" please

Reply:
We can probably use the same default as we use in our Web UI, "0000"

Reply:
Well.. Each and every product will have prolly some other default which we do not know and hence cannot accommodate upfront.